microsoft/iot-cardboard-js
# OAT: Disable adding arrays for V2 models

### Summary of changes 🔍 
When a model is DTDL V2, Arrays are not a supported property type. Likewise, if a relationship is being drawn from a model that is V2, the same logic applies.

![image](https://user-images.githubusercontent.com/57726991/217895240-ea03a716-bd2a-40b8-8130-e4c81f4d233c.png)


### Testing 🧪
- Try adding a property to a V2 model - see option disabled
- Try adding a property to a relationship that is from a V2 model - see option disabled
- Try changing a property type from something to array - see option disabled
